src/widgets/survey/ui/Finish.tsx

```diff
@@ -145,6 +150,14 @@ function FinishItem({
 
     const executionGroupKey = getExecutionGroupKey(progressRecord);
 
+    const logActivityName = getActivityName(activityId);
+
+    Logger.log(
+      `[Finish.completeActivity]: ${
+        flowId ? 'Flow' : 'Activity'
+      } ${logActivityName}|${activityId} completed`,
```

Comment:
It seems the message is incorrect. We'd get: Flow [activity info] completed. If you're talking about flow then should be: Activity [activity info] within Flow [flow info] completed

Reply:
And the next message - Flow [flow info] completed

Reply:
Consider this for Intermediate screen as well